docs/book/src/plugins/kustomize-v2-alpha.md

```diff
@@ -20,7 +21,7 @@ directory of the Kubebuilder project.
 ## When to use it
 
 - If you are looking to scaffold the kustomize configuration manifests for your own language plugin
-- If you are looking for support on Apple Silicon (`darwin/arm64`). (_Before kustomize `4.x` the binary for this plataform is not provided_)
+- If you are looking for support on Apple Silicon (`darwin/arm64`). (_Before kustomize `5.x` the binary for this plataform is not provided_)
 - If you are looking for to begin to try out the new syntax and features provide by kustomize v5
 - If you are NOT looking to build projects which will be used on Kubernetes cluster versions < `1.22` (_The new features provides by kustomize v4 are not officially supported and might not work with kubectl < `1.22`_)
```

Comment:
We need to check what k8s version is supported by kustomize v5

Reply:
in kustomize release note of v5 there is no mention of k8s version support. but from my research I found that: 
Kustomize v2.x.x is compatible with Kubernetes v1.9 and later.
Kustomize v3.x.x is compatible with Kubernetes v1.13 and later.
Kustomize v4.x.x is compatible with Kubernetes v1.16 and later.
Kustomize v5.x.x is compatible with Kubernetes v1.21 and later.

Reply:
let me know your reviews and I will try to update the file as suggested.